plugin/src/main/scala/org/apache/pekko/PekkoParadoxPlugin.scala

```diff
@@ -31,11 +31,15 @@ object PekkoParadoxPlugin extends AutoPlugin {
   object autoImport {
     val pekkoParadoxCopyright = settingKey[String]("Copyright text to use in docs footer")
     val pekkoParadoxGithub = settingKey[Option[String]]("Link to Github repository")
+    val pekkoParadoxIncubatorNotice = settingKey[Option[String]]("Whether to include the ASF incubator notice")
   }
   import autoImport._
 
   val version = ParadoxPlugin.readProperty("pekko-paradox.properties", "pekko.paradox.version")
 
+  val incubatorNoticeText =
```

Comment:
I put the text into a publicly accessible `val` so that if we set the default value of `pekkoParadoxIncubatorNotice` to `None` (which we will do in the future at some point) if for some reason someone wants to access the original text they can